README.md

```diff
@@ -51,5 +52,5 @@ out:
 ## Build
 
 ```
-$ rake
+$ ./gradlew package
```

Comment:
`$ ./gradlew gem  # -t to watch change of files and rebuild continuously` ?

Reply:
Ah yes, should be `./gradlew gem`. Thanks for catching